### PR TITLE
Update get-resources.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Added the complete example of getting resources for a apikey

### CODE REVIEW

File: docs/sdks/get-resources.md [hunk 1]
```diff
@@ -12,26 +12,65 @@ import TabItem from '@theme/TabItem';
...
 
 ```js
-//Fetch accessible databases
-try{
-    var databases = await client.listAccessibleDatabases(keyid)
-    console.log("Accessible Databases")
-    console.log(databases.result)
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/sdks/get-resources.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/get-resources.md [hunk 2]
```diff
@@ -12,26 +12,65 @@ import TabItem from '@theme/TabItem';
...
+const email = "nemo@nautilus.com";
+const password = "xxxxxx";
+const fabric = "_system";
+const keyid = "id1";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Create an authenticated instance with a JWT token.
```
Comment (diana-macrometa) [inline docs/sdks/get-resources.md:29]:
```suggestion
// Choose one of the following authentication methods and remove the commenting.

// Create an authenticated instance with a JWT token.
```

File: docs/sdks/get-resources.md [hunk 3]
```diff
@@ -12,26 +12,65 @@ import TabItem from '@theme/TabItem';
...
-    streams = await client.listAccessibleStreams(keyid, '_system', full=false)
-    console.log("Accessible Streams")
-    console.log(streams.result)
+async function main() {
+  await client
+    .login(email, password)
+    .then((e) => console.log("1. User authentication done!"))
+    .catch((error) => error);
+
+  console.log("\n2. Listing accessible Databases for Key_ID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/get-resources.md]:
```suggestion
  console.log("\n2. Listing accessible databases for Key_ID = " + keyid);
```

File: docs/sdks/get-resources.md [hunk 4]
```diff
@@ -12,26 +12,65 @@ import TabItem from '@theme/TabItem';
...
+  await client
+    .listAccessibleDatabases(keyid)
+    .then((databases) => {
+      console.log(databases.result);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n3. Listing accesible Streams for Key_ID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/get-resources.md]:
```suggestion
  console.log("\n3. Listing accessible streams for Key_ID = " + keyid);
```

File: docs/sdks/get-resources.md [hunk 5]
```diff
@@ -12,26 +12,65 @@ import TabItem from '@theme/TabItem';
...
+  await client
+    .listAccessibleStreams(keyid, fabric, (full = false))
+    .then((streams) => {
+      console.log(streams.result);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n4. Listing accesible Collections for Key_ID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/get-resources.md]:
```suggestion
  console.log("\n4. Listing accessible collections for Key_ID = " + keyid);
```

**Review by diana-macrometa — APPROVED** [review]
A few minor suggestions.

File: docs/sdks/get-resources.md [hunk 6]
```diff
@@ -12,26 +12,67 @@ import TabItem from '@theme/TabItem';
...
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Choose one of the following authentication methods and remove the commenting.
+
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/sdks/get-resources.md:31]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
One minor grammar change, otherwise looks good!

**Pull request merged**